Description: Drop tests erroring on RO db.
 These workaround test failures showing:
 .
   Doctrine\DBAL\Driver\PDO\Exception: SQLSTATE[HY000]:
   General error: 8 attempt to write a readonly database
Author: Bryce Harrington <bryce@canonical.com>
Origin: vendor
Forwarded: no; will do once the archive opens.
Last-Update: 2021-06-09

--- a/src/Symfony/Component/Messenger/Bridge/Doctrine/Tests/Transport/DoctrineIntegrationTest.php
+++ b/src/Symfony/Component/Messenger/Bridge/Doctrine/Tests/Transport/DoctrineIntegrationTest.php
@@ -48,6 +48,7 @@
 
     public function testConnectionSendAndGet()
     {
+        $this->markTestSkipped('Test requires read-write sqlite database');
         $this->connection->send('{"message": "Hi"}', ['type' => DummyMessage::class]);
         $encoded = $this->connection->get();
         $this->assertEquals('{"message": "Hi"}', $encoded['body']);
@@ -56,6 +57,7 @@
 
     public function testSendWithDelay()
     {
+        $this->markTestSkipped('Test requires read-write sqlite database');
         $this->connection->send('{"message": "Hi i am delayed"}', ['type' => DummyMessage::class], 600000);
 
         $stmt = $this->driverConnection->createQueryBuilder()
@@ -74,6 +76,7 @@
 
     public function testItRetrieveTheFirstAvailableMessage()
     {
+        $this->markTestSkipped('Test requires read-write sqlite database');
         $this->connection->setup();
         // insert messages
         // one currently handled
@@ -108,6 +111,7 @@
 
     public function testItCountMessages()
     {
+        $this->markTestSkipped('Test requires read-write sqlite database');
         $this->connection->setup();
         // insert messages
         // one currently handled
@@ -149,6 +153,7 @@
 
     public function testItRetrieveTheMessageThatIsOlderThanRedeliverTimeout()
     {
+        $this->markTestSkipped('Test requires read-write sqlite database');
         $this->connection->setup();
         $twoHoursAgo = new \DateTime('now');
         $twoHoursAgo->modify('-2 hours');
@@ -175,6 +180,7 @@
 
     public function testTheTransportIsSetupOnGet()
     {
+        $this->markTestSkipped('Test requires read-write sqlite database');
         $this->assertFalse($this->driverConnection->getSchemaManager()->tablesExist('messenger_messages'));
         $this->assertNull($this->connection->get());
 
